hknweb/admin.py

```diff
@@ -6,6 +6,10 @@
 
 
 
+from .models import Announcement
+MAX_STRLEN = 85
+MAX_TXTLEN = 2000
```

Comment:
Why are these needed here? I don't think they're used by anything in this file as far as I can tell